nix/latex-documents.nix

```diff
@@ -0,0 +1,146 @@
+{ repoRoot, inputs, pkgs, lib, ... }:
+
+let
+
+  latex-documents = lib.flip lib.mapAttrValues nix.plutus.build-latex-doc {
+
+    cost-model-notes = {
+      name = "cost-model-notes";
+      src = inputs.self + /doc/notes/cost-model-notes;
+      description = "Notes on cost models";
+      texFiles = [ "cost-model-notes.tex" ];
+    };
+
+    eutxo-paper = {
+      name = "eutxo-paper";
+      description = "eutxo";
+      src = inputs.self + /doc/papers/eutxo;
+      texFiles = [ "eutxo.tex" ];
+    };
+
+    eutxoma-paper = {
+      name = "eutxoma-paper";
+      description = "eutxoma";
+      src = inputs.self + /doc/papers/eutxoma;
+      texFiles = [ "eutxoma.tex" ];
+    };
+
+    extended-utxo-spec = {
+      name = "extended-utxo-spec";
+      src = inputs.self + /doc/extended-utxo-spec;
+      description = "Extended UTXO specification";
+    };
+
+    lazy-machine-notes = {
+      name = "lazy-machine-notes";
+      src = inputs.self + /doc/notes/fomega/lazy-machine;
+      texFiles = [ "lazy-plutus-core.tex" ];
+      description = "lazy machine discussion";
+    };
+
+    multi-currency-notes = {
+      name = "multi-currency-notes";
+      src = inputs.self + /doc/notes/multi-currency;
+      description = "Multi-currency paper";
+    };
+
+    plutus-core-spec-old = {
+      name = "plutus-core-spec-old";
+      description = "Plutus core specification (old version)";
+      src = inputs.self + /doc/plutus-core-spec-old;
+    };
+
+    plutus-core-spec = {
+      name = "plutus-core-spec";
+      description = "Plutus core specification";
+      src = inputs.self + /doc/plutus-core-spec;
+      texFiles = [ "plutus-core-specification.tex" ];
+    };
+
+    plutus-report = {
+      name = "plutus-report";
+      description = "plutus report";
+      src = inputs.self + /doc/plutus-report;
+      texFiles = [ "plutus.tex" ];
+    };
+
+    system-f-in-agda-paper = {
+      name = "system-f-in-agda-paper";
+      src = inputs.self + /doc/papers/system-f-in-agda;
+      description = "system-f in agda";
+      texFiles = [ "paper.tex" ];
+      withAgda = true;
+      agdaFile = "paper.lagda";
+    };
+
+    utxoma-paper = {
+      name = "utxoma-paper";
+      description = "utxoma";
+      src = inputs.self + /doc/papers/utxoma;
+      texFiles = [ "utxoma.tex" ];
+    };
+  };
+
+
+  unraveling-recursion-paper =
```

Comment:
can we put this in its own file? I kind of liked the std thing of having one derivation per file...